examples/metrics/prometheus-install/strimzi-pod-monitor.yaml

```diff
@@ -63,14 +63,14 @@ spec:
         values: ["Kafka", "KafkaConnect"]
   namespaceSelector:
     matchNames:
-      - myproject
+      - my-project
   podMetricsEndpoints:
   - path: /metrics
     port: tcp-prometheus
     relabelings:
     - separator: ;
-      regex: __meta_kubernetes_pod_label_(.+)
-      replacement: $1
+      regex: __meta_kubernetes_pod_label_strimzi_(.+)
+      replacement: strimzi_$1
```

Comment:
why adding the `strimzi_` prefix? the strimzi label matched by the regex should already have it. Doesn't it add the prefix twice? I haven't an installation with Prometheus handy, can you verify?

Reply:
without that the regex matches everything after strimzi_ and your metrics labels all begin with io_ instead of strimzi_io_... This could be cleaned up by just moving the matching group left parenthesis. I can do that if you think it is necessary.

Reply:
@brokenjacobs any update on this?